src/main/resources/index.jelly

```diff
@@ -1,4 +1,4 @@
 <?jelly escape-by-default='true'?>
 <div>
-    Plugin to pull in dependency on the Jackson parser data binding library (for JSON etc).
+    <b>Deprecated</b>. Plugin to pull in dependency on the Jackson parser data binding library (for JSON etc).
```

Comment:
Please, `<strong>` rather than `<b>`

Reply:
Too late, the release is running :(
Can do it next release, please create a pull request